dashboard/app/models/follower.rb

```diff
@@ -66,7 +66,7 @@ def assign_script
   def remove_family_name
     # If the student is in zero sections, and has a family name set,
     # remove the family name.
-    if student_user.family_name && student_user.sections_as_student.empty?
+    if student_user&.family_name && student_user&.sections_as_student&.empty?
```

Comment:
```suggestion
    return unless student_user
    
    if student_user.family_name && student_user.sections_as_student.empty?
```

Reply:
I like it. Seems cleaner. I dislike the prevalence of the `&.` syntax. I'm contemplating where the comment should go (does it move down to be near the logic it describes) or updating the comment to explain that, technically, `student_user` might be `nil` and why (for folks that might copy the pattern). For me: when in doubt, be verbose. Especially on a deletion code path. I'll make that change.

Reply:
The other option is to `return if student_user.deleted?` which is a tad more self described and specific. I might go that route.

Reply:
Belay that lol. Once I had some coffee I realized why that's a very silly idea. It's literally `nil`. I hate how deleted users act.